lte/gateway/c/core/oai/tasks/nas/esm/esm_pt.hpp

```diff
@@ -60,5 +59,3 @@ int esm_pt_is_reserved(int pti);
 #ifdef __cplusplus
```

Comment:
Can we move this fun() out of extern

Reply:
This will be done in a separate PR that I am working on